store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -233,6 +234,18 @@ public static void postCreateVolumeActions(CreateVolumeRequest request, Volume v
         }
     }
 
+    /**
+     * Append Server id in Volume prefix if Volume is not Unified
+     * @param volInfoRequest
+     * @param serverId
+     */
+    public static void appendServerIdInVolumePrefix(VolumeInfo volInfoRequest, String serverId) {
+        if (!volInfoRequest.getVolumeExternalInfo().isUnified()) {
+            volInfoRequest.getVolumeExternalInfo()
+                    .setVolumePrefix(volInfoRequest.getVolumeExternalInfo().getVolumePrefix() + "-" + serverId);
```

Comment:
Use ROOT_PATH_ELE_SEPARATOR instead of "-"

Reply:
done.